css/style.css

```diff
@@ -464,10 +496,270 @@ div .taggs {
   color: #fff;
 }
 
-.shape {
-  margin-left: 30%;
-  background: #0d151e;
-  border-radius: 10px;
-  width: 134px;
-  height: 5px;
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    background: #e5e5e5;
+  }
+
+  #main-page {
+    left: 0;
+    height: 800px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: #fff;
+    background-image: url(/images/desktop_main/Header.svg);
```

Comment:
- [ ] This image is not showing up in the browser. kindly consider using `.png` file. this background image should be there because we must stick to the figma design.
- [ ] Your design lack the bottom left rounded border in the main headline section background image.I will suggest you export the `white` background image from figma and set up in your design for better aesthetics.
your design
![qwer](https://user-images.githubusercontent.com/96683843/213526851-1dea1094-8baf-4abf-9e17-b538e836416c.jpg)

figma design

![qwe](https://user-images.githubusercontent.com/96683843/213526918-11e41c89-e64a-466e-82d6-cee0447fb9d2.jpg)